data-scripts/polisis.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs')
+const polisisData = require('../data/symlinked/polisis')
+const reasonsWithScore = {
+    "Certain data is shared with third parties for advertising purposes.": 4,
+    "Some personal information is shared with third parties.": 6,
+    "Data might be shared in the case of a merger or acquisition.": 1,
+    "Some data might be retained indefinitely.": 1
+}
+
+let outputData = {}
+
+Object.keys(polisisData).forEach((url) => {
+    outputData[url] = {
+        reasons: {
+            good: [],
+            bad: []
+        },
+        score: 0
+    }
+
+    Object.keys(polisisData[url].bad).forEach((reason) => {
+        if (reasonsWithScore[reason]) {
+            outputData[url].reasons.bad.push(reason)
+            outputData[url].score += reasonsWithScore[reason]
+        }
+    })
+})
```

Comment:
**Q:** I noticed a small thing about how Polisis data is obtained in the original prototype scripts: https://github.com/duckduckgo/privacy-grade/blob/9376fa575eb68f5323707160f6dc4c71d369fcdd/grade-scripts/post-process/grade.js#L383-L406

Note, in that snippet the privacy grade falls back to `polisis.bad`, which is obtained in an earlier step ([here](https://github.com/duckduckgo/privacy-grade/blob/9376fa575eb68f5323707160f6dc4c71d369fcdd/src/classes/grade.js#L42)), which includes all bad Polisis data points, even ones we might not be interested in.

I wasn't really sure if this was intentional. I've not implemented it here just yet.